Formula/k/ktor.rb

```diff
@@ -0,0 +1,26 @@
+class Ktor < Formula
+  desc "Generates Ktor projects through the command-line interface"
+  homepage "https://github.com/ktorio/ktor-cli"
+  url "https://github.com/ktorio/ktor-cli.git",
+      tag:      "0.2.0",
+      revision: "31bd486d11defbb573433dbb9f79f6e3e7ca407b"
```

Comment:
Do we need a Git clone here? Can we use a tarball?

Reply:
What's the problem with git clone?

Reply:
> What's the problem with git clone?

Git clones are slower and worse for caching, so we should avoid it unless we need it.

> Do you mean to use an URL like `https://github.com/ktorio/ktor-cli/archive/0.2.0.tar.gz` and `sha256`?

Yes, exactly.